easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.8.2-2023a.yml

```diff
@@ -46,3 +46,4 @@ easyconfigs:
   - ALL-0.9.2-foss-2023a.eb:
       options:
         from-pr: 19455
+  - ReFrame-4.3.3.eb
```

Comment:
@xinan1911 Since this easyconfig is not tied to a specific toolchain, it should be added to the `system` easystack file.

Ideally we also use EasyBuild v4.9.0, so use `eessi-2023.06-eb-4.9.0-001-system.yml` (the `001` there is just to make sure that the `system` easystack is considered first, before any toolchain-specific easystack file).

Reply:
Sounds good. I have added ReFrame v4.3.3 to eb v4.8.2 system and eb v4.9.0 system easystack.